modules/profiles/functions.php

```diff
@@ -3,10 +3,11 @@
 if (!defined('DEBUG_MODE')) { die(); }
 
 if (!hm_exists('add_profile')) {
-    function add_profile($name, $signature, $reply_to, $is_default, $email, $server_mail, $smtp_server_id, $imap_server_id, $context) {
+    function add_profile($name, $signature, $remark = '', $reply_to, $is_default, $email, $server_mail, $smtp_server_id, $imap_server_id, $context) {
```

Comment:
@Shadow243 $remark is defined here.

Reply:
It is defined here: https://github.com/cypht-org/cypht/pull/1190/files#r1727773754

Reply:
Yes but calling the fonction `add_profile` you are passing `$remark` but i can't see where it's coming from. You mean you can pass an undefined variable ?

Reply:
See even the VSCode 
<img width="955" alt="Screenshot 2024-08-22 at 23 34 20" src="https://github.com/user-attachments/assets/10a6742d-81fa-4416-9ceb-4db7aad67873">

Reply:
I'm, sure, it's defined on the form, you will get it from https://github.com/cypht-org/cypht/blob/master/modules/core/handler_modules.php#L990 and https://github.com/cypht-org/cypht/blob/master/modules/core/handler_modules.php#L1016

Reply:
> @Shadow243 $remark is defined here.

You can move param `$remark` at the end in the fonction definition, like that you dont need to pass it while calling. like that you will then remove it at line 1131.